### PR TITLE
Feature: Generate logos for various groups and sort into corresponding folders

### PR DESCRIPTION
**Pull request by g-voelker**
I have added support for the generation of logos for international chapters (countries), federal states, expert groups. These will be sorted into corresponding folders.

The merge request is ofc conditional on the resolution of
https://github.com/s4f-dortmund/regionalgruppen_logos/issues/31

### CODE REVIEW

**Comment by g-voelker** [comment]
I just realized that I pulled upstream updates from `master` instead of the new `main`. The conflict above relates to that.

**Comment by g-voelker** [comment]
All bugs resolved and checks fulfilled, should be fine now.

**Comment by g-voelker** [comment]
Shall we go forward with the pull request?

**Comment by g-voelker** [comment]
@maxnoe ? Are these repos orphaned? If so, shall I take over maintenance?

**Comment by maxnoe** [comment]
Hi @g-voelker sorry for not answering, this repeatedly slipped my mind.

File: bundeslaender.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Baden-Württemberg
+Bayern
+Berlin
+Brandenburg
+Bremen
+Hamburg
+Hessen
+Mecklenburg-Vorpommern
+Niedersachsen
+Dominikanische Republik
```
Comment (maxnoe) [inline bundeslaender.txt:10]: :rofl:

Comment (maxnoe) [reply bundeslaender.txt:10]: I was wondering why there were 17 lines in bundeslaender...

Comment (g-voelker) [reply bundeslaender.txt:10]: Haha. No idea how that ended up there. Thanks for spotting it.

**Pull request merged**